src/backend/rest.ts

```diff
@@ -58,13 +59,100 @@ type RestApiFilterRequest = {
   filter: unknown;
 } & CommonRequest;
 
+type RestApiSiteConnectionResponse = {
+  value: Array<{
+    extensions: {
+      basic_settings: {
+        alias: string;
+        site_id: string;
+      };
+    };
+  }>;
+};
+
+type RestApiHostResponse = {
+  value: Array<{
+    id: string;
+    extensions: {
+      name: string;
+    };
+  }>;
+};
+
+type RestApiServiceResponse = {
+  value: Array<{
+    id: string;
+    extensions: {
+      description: string;
+    };
+  }>;
+};
+
```

Comment:
Can we factor these together? I'm thinking of a `RestApiCollectionResponse<ExtenionType>` interface.